src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -808,6 +808,28 @@ this is given by McMullen's Upper-Bound-Theorem.
 """
 upper_bound_h_vector(d::Int,n::Int) = Vector{Int}(Polymake.polytope.upper_bound_theorem(d,n).H_VECTOR)
 
+@doc raw"""
+    billera_lee_polytope(h::AbstractVector)
+
+Construct a simplicial polytope whose h-vector is $h$.
+The corresponding g-vector must be an M-sequence.
+The ambient dimension equals the length of $h$, and the polytope lives in codimension one.
+- [BL81](@cite)
```

Comment:
I was trying to add a reference to the original paper of Billera and Lee.  Yet building the docs gave an error.  Help, please!

@lkastner

Reply:
I ran `bibtool` again due to https://github.com/oscar-system/Oscar.jl/actions/runs/4908035760/jobs/8763533061
It seems like you already ran it, so you just might have a different version?